mne/viz/ica.py

```diff
@@ -142,8 +143,19 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
     _plot_ica_topomap(ica, pick, show=False, axes=topo_ax, **topomap_args)
 
     # image and erp
+    # we create a new epoch with dropped rows
+    epoch_data = epochs_src.get_data()
+    epoch_data = np.insert(epoch_data,
```

Comment:
I would use the paramters of insert to document what is happening (see later in the code I give you an example)